PointerStar/Client/Pages/Room.razor

```diff
@@ -173,7 +233,27 @@
         await base.OnInitializedAsync();
         if (RoomId is { } roomId)
         {
-            User user = new(Guid.NewGuid(), Name ?? $"User {new Random().Next()}");
+            if(string.IsNullOrEmpty(Name))
```

Comment:
Should this be moved into the ViewModel? There is an `OnInitializedAsync` method that can be overriden there.